libraries/webInterface/gui/index.js

```diff
@@ -262,6 +388,33 @@ realityServer.updateManageObjects = function (thisItem2) {
                 thisObject.dom.querySelector('.target').setAttribute('isWorldObject', true);
                 thisObject.dom.querySelector('.target').addEventListener('click', realityServer.gotClick, false);
 
+                setTooltipTextForElement('.name',
```

Comment:
There's multiple things with class name so the setTooltipTextForElement function should use querySelectorAll to give them all tooltips